bin/jwtwebhook.py

```diff
@@ -26,23 +26,24 @@
 from splunk_helper import data_encryption
 from modular_input import ModularInput, Field, IntegerField, FilePathField
 from webhooks_input_app.flatten import flatten
+from splunklib.modularinput.event_writer import EventWriter
+from splunklib.modularinput.event import Event
```

Comment:
If we're introducing splunklib's version here... do we want to rework and get rid of the custom modular_input library as well?

Reply:
Completed